### ISSUE TITLE
Kubevirt application Health is displayed as Idle

### ISSUE THREAD

**Issue by rhrazdil** [kiali/kiali#3734]
**Describe the bug**
THe VirtualMachineInstance is a KubeVirt resource representing a running virtual machine. 
It has an pod associated to it, called the virt-launcher-*, which is the actual VM.
Kiali should be able to pick it up and display it as ready, when the virt-launcher pod exists and is ready

Apparently, the problem is that kiali expects there to be a pod called vmi-cirros(?), but that is not the case, the 
pod name for a virtualMachineInstance is `virt-launcher-<VMI name>-<random>`

For UI issues please also add a screenshot that shows the issue.
![Screenshot from 2021-02-24 20-50-55](https://user-images.githubusercontent.com/32546791/109059519-9fc45f80-76e4-11eb-92b2-aa97e1a94286.png)

**Versions used**
Kiali: v1.29.1
Istio: 1.9.0
Kubernetes flavour and version: k8s v1.20.2

**To Reproduce**
0. Have istio deployed and the current namespace labeled to auto-inject
1. Deploy KubeVirt
```
export RELEASE=v0.37.0
kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt-operator.yaml 
kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt-cr.yaml
```
2. Create VMI
```
kubectl create -f https://raw.githubusercontent.com/rhrazdil/istio_samples/master/virtualmachines/vmi-cirros
```
3. Go to Kiali dashboard
4. Go to Applications, see the vmi-cirros Health Status

**Expected behavior**
Status should be Healthy, as the underlying pod is running and ready

**Comment by lucasponce** [comment]
Yeah, it's related to the previous issue https://github.com/kiali/kiali/issues/3733

These workloads have a special controller that would need to query to get details of workload status.

**Comment by lucasponce** [comment]
I think this is a consequence of the https://github.com/kiali/kiali/issues/3733 as the controller is not correctly pulled, the health information for this custom controller is wrong.

So this issue could be closed in favor of #3733
